x-pack/plugins/ml/public/components/controls/select_interval/__tests__/select_interval_directive.js

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from 'expect.js';
+import ngMock from 'ng_mock';
+
+import { interval$ } from '../select_interval';
+
+describe('ML - mlSelectIntervalService', () => {
+  let appState;
+
+  beforeEach(ngMock.module('kibana', (stateManagementConfigProvider) => {
+    stateManagementConfigProvider.enable();
+  }));
+  beforeEach(ngMock.module(($provide) => {
+    appState = {
+      fetch: () => {},
+      save: () => {}
+    };
+
+    $provide.factory('AppState', function () {
```

Comment:
nit, there are a few functions which could be arrow functions

Reply:
Done in 7e2859b.